Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -2,14 +2,56 @@
 {
     public static class ArrayOperations
     {
+
+        static bool IsArraySorted(int[] array)
+        {
+            for (int i = 0; i < array.Length + 1; i++)
+            {
+                //This to stop IndexOutOfRangeException
+                if (i + 1 >= array.Length)
+                {
+                    break;
+                }
```

Comment:
With the above suggestion this code should be obsolete. Also you should not go for such hacky solutions.